bftengine/src/bftengine/SeqNumInfo.cpp

```diff
@@ -40,6 +40,13 @@ SeqNumInfo::~SeqNumInfo() {
 }
 
 void SeqNumInfo::acquire(SeqNumInfo& rhs) {
+  primary = rhs.primary;
```

Comment:
how could it work before?

Reply:
It was not working correctly. It was only working in the case where a Replica has actually received the PrePrepare from the Primary. With the fix the Primary will resend the PrePrepares on Status exchange.